open_fdd/validation/bench_5007_long_fdd.py

```
@@ -0,0 +1,836 @@
+def run_synthetic_validation(cfg: SmokeConfig, model: dict[str, Any] | None = None) -> ValidationReport:
+    report = ValidationReport(
+        config=cfg,
+        started_at=datetime.now(timezone.utc).isoformat(),
+        environment={"mode": "synthetic", "datafusion_installed": datafusion_available()},
+    )
+    if model is None:
+        model = json.loads((Path(__file__).resolve().parents[2] / "workspace/data/bench_dual_source_model.json").read_text())
+
+    report.errors.extend(validate_model_preflight(model, cfg))
+    aligned = align_semantic_points(model, cfg.site_id)
+    for semantic, by_source in aligned.items():
```

Ruff (0.15.15)

[warning] Loop control variable `semantic` not used within loop body

Rename unused `semantic` to `_semantic`

(B007)